nutrinova-api/NutrinovaData.UnitTests/Features/Recipes/RecipeFoodTotalerSteps.cs

```diff
@@ -69,14 +69,16 @@ public void GivenTheFollowingFoodNutrients(Table table)
     var foodNutrients = table.CreateSet<FoodPlanNutrient>().ToList();
     var foods = this.scenarioContext.Get<List<FoodPlan>>("Foods");
     var nutrients = this.scenarioContext.Get<List<Nutrient>>("Nutrients");
+    var units = this.scenarioContext.Get<List<Unit>>("Units");
 
     foreach (var foodNutrient in foodNutrients)
     {
       foodNutrient.Foodplan = foods?.FirstOrDefault(f => f.Id == foodNutrient.FoodplanId) ?? new FoodPlan { Id = foodNutrient.FoodplanId };
       foodNutrient.Nutrient = nutrients?.FirstOrDefault(n => n.Id == foodNutrient.NutrientId) ?? new Nutrient { Id = foodNutrient.NutrientId };
+      foodNutrient.Unit = units?.FirstOrDefault(u => u.Id == foodNutrient.UnitId) ?? new Unit { Id = foodNutrient.UnitId };
     }
 
-    foreach (var food in foods)
+    foreach (var food in foods!)
```

Comment:
What does that ! Do?

Reply:
A useless null check, I should probably take it out